index.html

```diff
@@ -1,255 +1,373 @@
-			 <h3 class="languagesn">Languages</h3>
-			 <i class="arrow down"></i>
-		   </div>
-	 
-		   <ul class="languages-list">
-			 <li class="languages-Icon"><img class="Icons" src="Images/Js.icon.png" alt="JavaScript">
-				<p class="name">javascript</p></li>
-			
-			 <li class="languages-Icon"><img class="Icons" src="Images/Html.icon.png" alt="HTML">
-				<p class="name">HTML</p></li>
-			
-			 <li class="languages-Icon"><img class="Icons" src="Images/Css.icon.png" alt="CSS">
-				<p class="name">CSS</p></li>
-		   </ul>
-	 
-		   <div class="languages line">
-			 <h3 class="languagesn">Frameworks</h3>
-			 <i class="arrow right"></i>
-		   </div>
-
-		   <div class="languages line">
-			 <h3 class="languagesn">Skills</h3>
-			 <i class="arrow right"></i>
-		   </div>
-		 </div>
-		</section>
-		<section class="Myform">
-			<div class="form-container">
-			  <h2 class="contact-title">Contact me</h2>
-			  <p class="contact-text">If you have an application you are
-				 interested in developing, a feature that you need built 
-				 or a project that needs coding. I'd love to help with it </p>
-			  <div class="enter">
-				<form action="https://formspree.io/f/xbjbnvvz" method="POST">
-				  <input class="entername" type="text" 
-				  placeholder="Enter your name here" maxlength="30" required>
-				  <input class="entername" type="email" 
-				  placeholder="Enter your Email here" required>
-				  <textarea class="entermessage" name="area" cols="30" rows="6"  maxlength="500" 
-				  placeholder="Write your message here" required></textarea>
-				  <button class="get-in-touch" type="submit">Get in touch</button>
-				</form>
-			  </div>
-			</div>
-		</section>
-</body>
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+    <script
+      src="https://kit.fontawesome.com/27ebd9ab14.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <section class="header">
+      <div class="header-page">
+        <a href="#" class="Mylogo"> GDM Services</a>
+        <a class="Hamberger"><i class="fa-solid fa-bars"></i></a>
+
+        <ul class="option">
+          <li><a href="#portfolio">Portfolio</a></li>
+          <li><a href="#about">About</a></li>
+          <li><a href="#contact">Contact</a></li>
+        </ul>
+      </div>
+
+      <div class="header-container">
+        <h1 class="title1">I'm Gardimy. <br />Glad to see you!</h1>
+        <p class="title2">
+          I'm a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don't hestiate to contact
+          me.
+        </p>
+
+        <div class="social-icon">
+          <h2 class="connect" href="#">LET'S CONNECT</h2>
```

Comment:
- I noticed that your website still has some contrast issues kindly fix them:
![Screenshot from 2023-04-04 20-12-00](https://user-images.githubusercontent.com/97834160/229881165-95195549-3348-488f-9826-795129f2713b.png)
![Screenshot from 2023-04-04 20-13-02](https://user-images.githubusercontent.com/97834160/229881425-08f1a7f9-89a9-4675-a138-c61ddcd228ee.png)
__Hint: You can use this [link](https://color.a11y.com/?wc3) to test your contrast ratio.__